.github/workflows/ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
         run: |
           tox ${{ needs.check_mturk_changes.outputs.needs_mturk_tests == 'true' && '-- --mturkfull' || ''}}
           npm run test --coverage
-        if: matrix.python-version == 3.10
+        if: matrix.python-version == 3.10 || matrix.python-version == 3.11
       - name: Run Fast Tests Only
```

Comment:
I think the idea is that we only run MTurk tests once, so I don't think we need it on both 3.10 and 3.11.

Reply:
Running MTurk tests on Python 3.11 only now